apollo-router/src/plugins/cache/entity.rs

```diff
@@ -518,23 +523,20 @@ impl Service<subgraph::Request> for CacheService {
 
     fn poll_ready(
         &mut self,
-        cx: &mut std::task::Context<'_>,
+        _cx: &mut std::task::Context<'_>,
     ) -> std::task::Poll<Result<(), Self::Error>> {
-        match &mut self.0 {
-            Some(s) => s.service.poll_ready(cx),
-            None => panic!("service should have been called only once"),
-        }
+        Poll::Ready(Ok(()))
```

Comment:
Should be calling `service.poll_ready`

Reply:
There is no `service` to call here, since this is the end of the chain. This is correct if you look at what the call function does.

It basically clones itself, then uses the clone to drive the service.

Reply:
Ignore my comment. I fixed this: 017b0f1e3fc88855b14ddcfc513fa2415bedbc11